src/app/jobs/weatherevents.job.ts

```diff
@@ -0,0 +1,51 @@
+import { Job } from '../../common/job';
+import { IContainer } from '../../common/types';
+import { TextChannel } from 'discord.js';
+
+interface IWeather_Event {
+  features?: (IFeaturesEntity)[] | null;
+}
+
+interface IFeaturesEntity {
+  properties: IProperties;
+}
+
+interface IProperties {
+  severity: string;
+  headline: string;
+  senderName: string;
+}
+
+export class WeatherEventsJob extends Job {
+  // Run once an hour
+  public interval: number = 1000 * 60 * 60;
+
+  public name: string = 'weather_events';
+
+  constructor() {
+    super();
+  }
+
+  public async execute(container: IContainer) {
+    const channel = (container.guildService.getChannel("weather_events")) as TextChannel;
+    const resp = await container.httpService.get("https://api.weather.gov/alerts/active?area=FL");

```

Comment:
These should be single quotes, do you have `prettier` installed?

Reply:
I do, but for some reason, it's not formatting on save for me.